projects/canopy/src/styles/variables/_main.scss

```diff
@@ -13,6 +13,7 @@
   --z-index-modal-overlay: 1000;
   --z-index-modal: 1010;
   --z-index-carousel-autoplay: 1;
+  --z-index-skip-link: 9999;
```

Comment:
Do you think this should be a more considered value like `1020` and then if we need it to be greater in our app because we have higher z-indexes in our custom nav, then we could override this to suit our needs

Reply:
Updated